plugins/inputs/jolokia2/examples/kafka-connect.conf

```diff
@@ -0,0 +1,94 @@
+    name         = "taskErrorMetrics"
+    mbean        = "kafka.connect:type=task-error-metrics,connector=*,task=*"
+    paths        = ["deadletterqueue-produce-failures", "deadletterqueue-produce-requests", "last-error-timestamp", "total-errors-logged", "total-record-errors", "total-record-failures", "total-records-skipped", "total-retries"]
+    tag_keys     = ["connector", "task"]
+
+  # https://kafka.apache.org/documentation/#selector_monitoring
+  [[inputs.jolokia2_agent.metric]]
+    name         = "connectMetrics"
+    mbean        = "kafka.connect:type=connect-metrics,client-id=*"
+    paths        = ["connection-close-rate", "connection-close-total", "connection-creation-rate", "connection-creation-total", "network-io-rate", "network-io-total", "outgoing-byte-rate", "outgoing-byte-total", "request-rate", "request-total", "request-size-avg", "request-size-max", "incoming-byte-rate", "incoming-byte-rate", "incoming-byte-total", "response-rate", "response-total", "select-rate", "select-total", "io-wait-time-ns-avg", "io-wait-ratio", "io-time-ns-avg", "io-ratio", "connection-count", "successful-authentication-rate", "successful-authentication-total", "failed-authentication-rate", "failed-authentication-total", "successful-reauthentication-rate", "successful-reauthentication-total", "reauthentication-latency-max", "reauthentication-latency-avg", "failed-reauthentication-rate", "failed-reauthentication-total", "successful-authentication-no-reauth-total"]
+    tag_keys     = ["client-id"]
+
+  # https://kafka.apache.org/documentation/#common_node_monitoring
+  [[inputs.jolokia2_agent.metric]]
+    name         = "connectNodeMetrics"
+    mbean        = "kafka.connect:type=connect-node-metrics,client-id=*,node-id=*"
+    paths        = ["outgoing-byte-rate", "outgoing-byte-total", "request-rate", "request-total", "request-size-avg", "request-size-max", "incoming-byte-rate", "incoming-byte-total", "request-latency-avg", "request-latency-max", "response-rate", "response-total"]
+    tag_keys     = ["client-id", "node-id"]
+
+  [[inputs.jolokia2_agent.metric]]
+    name         = "appInfo"
+    mbean        = "kafka.connect:type=app-info,client-id=*"
+    paths        = ["start-time-ms", "commit-id", "version"]
+    tag_keys     = ["client-id"]
+
+  [[inputs.jolokia2_agent.metric]]
+    name         = "connectCoordinatorMetrics"
+    mbean        = "kafka.connect:type=connect-coordinator-metrics,client-id=*"
+    paths        = ["join-time-max", "failed-rebalance-rate-per-hour", "rebalance-latency-total", "sync-time-avg", "join-rate", "sync-rate", "failed-rebalance-total", "rebalance-total", "last-heartbeat-seconds-ago", "heartbeat-rate", "join-time-avg", "sync-total", "rebalance-latency-max", "sync-time-max", "last-rebalance-seconds-ago", "rebalance-rate-per-hour", "assigned-connectors", "heartbeat-total", "assigned-tasks", "heartbeat-response-time-max", "rebalance-latency-avg", "join-total"]
+    tag_keys     = ["client-id"]
+
+
+
+
```

Comment:
Minor nitpick. Can you delete these new lines? Thank you!

Reply:
Thank you :)